qdrant-landing/content/documentation/cloud/backups.md

```diff
@@ -1,36 +1,95 @@
 ---
 title: Backups
-weight: 30
+weight: 60
 ---
 
-# Backups
+# Cloud Backups
 
-There are situations where you need to restore your cluster because of application or system failure.
-In most cases you will have a source of truth for your data in a regular database and would be able to reindex the data into your Qdrant vector search cluster.
-However, encoding and uploading a big amount of data might require a long time.
-For high availability critical projects we highly recommend relying on replication, which is always a better option, because it guarantees the proper cluster functionality as long as at least one replica is running.
-For less critical use-cases you can make use of one of the available options.
+Qdrant organizes cloud instances as clusters. On occasion, you may need to
+restore your cluster because of application or system failure.
+
+You may already have a source of truth for your data in a regular database. If you
+have a problem, you could reindex the data into your Qdrant vector search cluster.
+However, this process can take time. For high availability critical projects we
+recommend replication. It guarantees the proper cluster functionality as long as
+at least one replica is running.
+
+For less critical use-cases you can set up automatic or self-service backups.
```

Comment:
It's not only for less critical use-cases. It's also necessary for disaster recovery.

Reply:
Corrected in the [new PR](https://github.com/qdrant/landing_page/pull/585/files#diff-4682bb6a3b58fe427a5ab171f794a7499ba5d95088d01bcc2a9e819784c90ab6R17-R18)